test-framework/core/src/main/java/org/keycloak/testframework/events/AdminEventAssertion.java

```diff
@@ -82,6 +82,11 @@ public AdminEventAssertion resourcePath(String... expectedResourcePath) {
         return this;
     }
 
+    public AdminEventAssertion resourcePath(String expectedResourcePath) {
+        Assertions.assertEquals(expectedResourcePath, event.getResourcePath());
+        return this;
+    }
+
```

Comment:
This is not needed, and you can call resourcePath(String... expectedResourcePath) with a single string
```suggestion
```

Reply:
ok

Reply:
I put together some conditional check, but overloading seems to me more nicer

Reply:
updated, keeping original version